frontend/pages/shopping-lists/_id.vue

```diff
@@ -792,15 +792,7 @@ export default defineComponent({
       itemsByLabel.value = itemsSorted;
     }
 
-    async function refreshLabels() {
-      const { data } = await userApi.multiPurposeLabels.getAll();
-
-      if (data) {
-        allLabels.value = data.items ?? [];
-      }
-    }
-
-    refreshLabels();
+    labelActions.getAll()
```

Comment:
Actually, do we even need this here at all? Can we just use the store without calling refresh?

Originally I had thought that we used the refresh function elsewhere, but upon further review it looks like we only use it here anyway

Reply:
you are correct. removing the call here has the list sorted alphabetically as expected.

it makes sense that this was superfluous now. i saw two requests in the network tab prior to this change, one with and without the ordering query params. i could never track down where the two calling locations were. but it makes sense that one was the original store request and the other was this one.

TL;DR removed

Reply:
Sweet. Yeah that file is a bit of a mess, largely my fault